martypy/Marty.py

```diff
@@ -591,11 +591,15 @@ def get_joint_status(self, joint_name_or_num: Union[int, str]) -> int:
             jointIDNo = self.JOINT_IDS.get(joint_name_or_num, 0)
         return self.client.get_joint_status(jointIDNo)
 
-    def get_distance_sensor(self) -> float:
+    def get_distance_sensor(self) -> Union[int, float]:
         '''
         Get the latest value from the distance sensor :one: :two:
         Returns:
-            The distance sensor reading (will return 0 if no distance sensor is found)
+            The distance sensor reading. The meaning of the returned value is different 
+            between Marty V1 and V2:
+                - :one: Returns a raw distance sensor reading as a 'float'.
+                - :two: Returns the distance in millimeters as 'int'.
```

Comment:
```suggestion
                - :one: Returns a raw distance sensor reading as a `float`.
                - :two: Returns the distance in millimeters as `int`.
```
(This way, it will be formatted `as code`.)